tests/components/picnic/test_todo.py

```diff
@@ -0,0 +1,90 @@
+"""Tests for Google Tasks todo platform."""
+
+
+import copy
+import json
+import unittest
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components.picnic import const
+from homeassistant.components.picnic.const import CONF_COUNTRY_CODE
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.helpers import entity_registry as er
+
+from tests.common import MockConfigEntry, async_test_home_assistant, load_fixture
+
+CART_FULL_RESPONSE = json.loads(load_fixture("picnic/cart.json"))
+CART_NO_RESPONSE = {}
+CART_EMPTY_RESPONSE = {
+    "items": [],
+}
+
+
+@pytest.mark.usefixtures("hass_storage")
+class TestPicnicSensor(unittest.IsolatedAsyncioTestCase):
```

Comment:
We don't accept any test written with `unittest`. Please see the pytest style instead

Reply:
Just used the same test structure as used in sensor.py to be consistent

I will see if I can convert the the other format.

Reply:
Test in test_todo.py have been updated to pytest style. Tests in test_sensor.py can be done in a followup PR.